### PR TITLE
implement hover on header&footer

### PR DESCRIPTION
**Pull request by robert-burcea**
Issue: Hover effect missing #339

- [ ] Added hover effect for "Alătură-te" header button
- [ ] Added hover effect for social media icons - footer

https://github.com/peviitor-ro/search-engine/assets/28507505/146cc289-afe2-4d1c-9288-c9d3bbfd274f

### CODE REVIEW

File: src/components/header/topbar.style.scss [hunk 1]
```diff
@@ -24,10 +24,23 @@
...
+
+  .logo-href {
+    padding: 8px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &:hover {
+      background-color: #f9e8d3;
+      border-radius: 5px;
```
Comment (eBitzu) [inline src/components/header/topbar.style.scss:36]: border radius pe hover? why?

**Review by sebiboga — APPROVED** [review]

File: src/components/header/topbar.style.scss (hunk 1 above)
Comment (robert-burcea) [reply src/components/header/topbar.style.scss:36]: Pentru ca unii dintre noi suntem la început și încă învățăm. Deci era mai bine lăsat în clasa border-radius-ul. Got it :) thanks

Comment (eBitzu) [reply src/components/header/topbar.style.scss:36]: poi, te rog modifica :)

**Comment by robert-burcea** [comment]
Yes, please cancel it. I will redo it on the updated main taking into consideration what @eBitzu told me to do. I apologise for leaving this issue hanging, have been away for more than a week, back in full throttle now. Thank you!

**Pull request closed without merging**